prime-router/src/main/kotlin/cli/LivdTableDownload.kt

```diff
@@ -41,33 +43,79 @@ private const val sheetName = "LOINC Mapping"
-	If the option is not specified, it will download the file to ./build directory.
+    It looks for the LIVD-SAR-CoV-2-yyyy-MM-dd.xlsx file from $cdcLOINCTestCodeMappingPageUrl.  If the file is found, 
+    it downloads the file into the ./build directory.  If not found, it will prompt error accordingly.  Next, it build 
+    the output Lookup Table (<tableName.csv> file) with the table name specified by --output-file <path/filename> 
+    option. If the --output-file is not specified, it will use the ./metadata/tables/$livdSARSCov2File-latest.csv as
+    the default output file.  Finally, it uses the output filename as the new lookup table name to update the 
+    lookup tables in the database as the new version of the table.  It updates new version of the lookup table in 
+    the given --env [local, test, staging, or prod] with the default to "local" environment.
+    
+    Note, this command will always create new version of the lookup table.
         
     Example: The following command will download the latest LIVD-SARS-CoV-2-yyyy-MM-dd.xlsx from the above URL.  
-      It will store the LIVD-SARS-CoV-2-xyz.csv file under the ./junk directory.
+      It will store the LIVD-SARS-CoV-2-xyz.csv file under the ./junk directory and update the LIVD-SARS-CoV-2-xyz
+      lookup table in the database.
       
          ./prime livd-table-download --output-file ./junk/LIVD-SARS-CoV-2-xyz.csv
+         
+      The command below will download the latest LIVD-SARS-CoV-2-yyyy-MM-dd.xlsx file from the above UTL.
+      It will crate the LIVD-SARS-CoV-2-latest.csv file under the ./metadata/tables directory.
+      
+        ./prime livd-table-download
     """
 ) {
     private val defaultOutputDir = "./build"
+    private val defaultOutPutFile = "./metadata/tables/$livdSARSCov2File-latest.csv"
     private val outputFile by option(
         "--output-file",
         metavar = "<path/filename>",
         help = "Output pathname where to store the LIVD data as CSV."
-    ).required()
+    ).default(defaultOutPutFile)
+
+    /**
+     * The environment to connect to.
+     */
+    private val env by option(
+        "-e", "--env",
+        metavar = "<name>",
+        envvar = "PRIME_ENVIRONMENT",
+        help = "Connect to <name> environment.\nChoose between [local|test|staging|prod]"
+    )
+        .choice(
+            Environment.LOCAL.envName, Environment.STAGING.envName,
+            Environment.TEST.envName, Environment.PROD.envName
+        )
+        .default(Environment.LOCAL.envName, "local environment")
+
+    /**
+     * The environment the command needs to run on.
+     */
+    internal val environment get() = Environment.get(env)
+
+    /**
+     * The reference to the table creator command.
+     */
+    private val tableCreator = LookupTableCreateCommand()
 
     override fun run() {
+        // Download the LIVD-SARS-CoV2-yyyyMMdd.xlsx from CDC web site given above.
```

Comment:
Add a message at the very top just stating that we are downloading the file.  I am thinking this will give an indication the process started in case it gets hung with the download

Reply:
Done.